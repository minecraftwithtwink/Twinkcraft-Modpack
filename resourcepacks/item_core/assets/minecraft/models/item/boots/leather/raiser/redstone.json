{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/leather/raiser",
        "layer1": "minecraft:item/boots/base/leather_overlay/raiser",
        "layer2": "minecraft:trims/items/boots_trim/raiser_redstone"
    }
}