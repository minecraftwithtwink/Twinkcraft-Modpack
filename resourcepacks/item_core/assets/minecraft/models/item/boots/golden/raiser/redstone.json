{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/golden/raiser",
        "layer1": "minecraft:trims/items/boots_trim/raiser_redstone"
    }
}